Specimen/Specimen-SolidTumorDemeizaSeo-Example.json

```diff
@@ -0,0 +1,78 @@
+{
+  "resourceType": "Specimen",
+  "id": "Specimen-SolidTumorDemeizaSeo-Example",
+  "extension": [
+    {
+      "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-SampleCategory",
+      "valueCodeableConcept": {
+        "coding": [
+          {
+            "system": "https://fhir.hl7.org.uk/CodeSystem/UKCore-SampleCategory",
+            "code": "tumour",
```

Comment:
CodeSystem now has a code for solid-tumour: https://simplifier.net/hl7fhirukcorer4/codesystem-ukcore-samplecategory